content/html/concepts/tags/terms/basefont/basefont.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '<basefont>'
+Description: 'Used to be used to set the font of text. This is now deprecated.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Style'
+  - 'Typography'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<basefont>`** tag was used to set the default font characteristics for the text of an HTML document. Since it is now deprecated, [CSS](https://www.codecademy.com/resources/docs/css)  properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead. To set the font of specific parts of the webpage the [`<font>`](https://www.codecademy.com/resources/docs/html/tags/font) would be used in combination with the `<basefont>` tag.

```

Comment:
```suggestion
The **`<basefont>`** tag was used to set the default font characteristics for the text of an HTML document. Since it is now deprecated, [CSS](https://www.codecademy.com/resources/docs/css)  properties like the [`font`](https://www.codecademy.com/resources/docs/css/typography/font) property should be used instead. To set the font of specific parts of the webpage the `<font>` tag would be used in combination with the `<basefont>` tag.

```

No need to have duplicate hyperlinks to the same content.

Reply:
The first was the font CSS property and this is the HTML font tag.

Reply:
Ah, I overlooked! For better clarity lets use the following edit:

```suggestion
The **`<basefont>`** tag was used to set the default font characteristics for the text of an HTML document. Since it is now deprecated, [CSS](https://www.codecademy.com/resources/docs/css)  properties like the [`font` property](https://www.codecademy.com/resources/docs/css/typography/font) should be used instead. To set the font of specific parts of the webpage the [`<font>` tag](https://www.codecademy.com/resources/docs/html/tags/font) would be used in combination with the `<basefont>` tag.
```